scipy/stats/stats.py

```diff
@@ -2492,7 +2492,7 @@ def sigmaclip(a, low=4., high=4.):
     return SigmaclipResult(c, critlower, critupper)
 
 
-def trimboth(a, proportiontocut, axis=0):
+def trimboth(a, proportiontocut=0.2, axis=0):
```

Comment:
This one is tricky, because it doesn't really make sense to have a default here. I'd rather drop the default from the mstats version, even though that's backwards-incompatible.